v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/rowmapper/JdbcSourceRowMapperTest.java

```diff
@@ -240,7 +240,7 @@ private static ImmutableList<Col> getTestCols() {
                 .build())
         .add(
             Col.builder("blob_col", "BLOB", new SourceColumnType("BLOB", new Long[] {10L}, null))
-                .withValue(new byte[] {0x65, 0x66}, new char[] {'6', '5', '6', '6'})
+                .withValue(new byte[] {0x65, 0x66}, "6566")
```

Comment:
Thanks.
I have added a null case for all the types.
A few other edge cases are covered in the exception test cases.

We are not checking with a real DB in the UT, so it might not be possible to test with the limits of blob length, would cover that in integration tests.